pageserver/src/tenant/ephemeral_file.rs

```diff
@@ -367,8 +368,10 @@ mod tests {
     }
 
     #[test]
-    fn test_ephemeral_files() -> Result<(), io::Error> {
-        let (conf, tenant_id, timeline_id) = harness("ephemeral_files")?;
+    fn test_ephemeral_files() -> io::Result<()> {
+        let tmp_dir = tempfile::tempdir()?;
+        let repo_dir = tmp_dir.path().join("ephemeral_files");
```

Comment:
Can we just used `tempfile::tempdir()` inside `harness()`, thereby avoiding repetition of the test name here?

Same for `test_ephemeral_blobs`

Reply:
This also should eliminate `!repo_dir.exists()` check

Reply:
Reused the harness, thanks.